source/includes/backups/_10-create.md.erb

```diff
@@ -34,39 +52,41 @@ Content-Type: <%= data.apis.versions.backup %>; charset=utf-8
       },
       "find": {
         "href": "https://ci.example.com/go/api/users/:login_name"
+      },
+      "current_user" : {
+        "href" : "https://ci.example.com/go/api/users/current_user"
       }
     },
     "login_name": "username"
   }
 }
 ```
 
-Create a server backup.
-
 <aside class="warning">
   The server may be unavailable during the time that the backup is being taken.
 </aside>
 
-<aside class="notice">
-  The response from the call will not be received till the backup is complete
-  (synchronous). If the data to be backed up is a lot, this can take minutes
-  to complete.
-</aside>
-
-
-<%=
-available_since('15.2.0') do
+<%= available_since('19.3.0') do
   in_version '15.1.0', url: 'https://www.gocd.org/documentation/user/15.1.0/api/backup_api.html'
   in_version '14.4.0', url: 'https://www.gocd.org/documentation/user/14.4.0/api/backup_api.html'
   in_version '14.3.0', url: 'https://www.gocd.org/documentation/user/14.3.0/api/backup_api.html'
 end
 %>
 
-
 <p class='http-request-heading'>HTTP Request</p>
 
 `POST /go/api/backups`
 
-<p class='http-request-return-description'>Returns</p>
+<p class='http-request-return-description'>
+  Returns an empty response body with a response code 202 (Accepted) and location header containing the link to be polled for checking the status of the backup.
+</p>
+
+`GET /go/api/backups/:scheduled_backup_id`
```

Comment:
I think the GET endpoint should be in a sidebar heading of its own, since it's a separate endpoint.

I would also prefer to call the parameter `:backup_id` instead of `:scheduled_backup_id`, since we can use an ID to also GET older backups.

Reply:
Comment incorporated.